exercises/binary-search/src/example/java/BinarySearch.java

```diff
@@ -0,0 +1,70 @@
+
+import java.util.List;
+
+public class BinarySearch<T extends Comparable<T>> {
+
+    public static String ARRAY_MUST_BE_SORTED = "Array should be sorted.";
```

Comment:
Sorry, missed this. Static analyzers had several other complaints including Javadocs, so didn't spot it. Are Javadocs actually allowed in the project?